README.md

```diff
@@ -55,7 +55,7 @@ We use the terms *study* and *trial* as follows:
 - Trial: a single execution of the objective function
 
 Please refer to sample code below. The goal of a *study* is to find out the optimal set of
-hyperparameter values (e.g., `classifier` and `svm_c`) through multiple *trials* (e.g.,
+hyperparameter values (e.g., `regressor` and `svm_c`) through multiple *trials* (e.g.,
```

Comment:
How about updating the hyperparameter name of `SVR` as well?
```suggestion
hyperparameter values (e.g., `regressor` and `svr_c`) through multiple *trials* (e.g.,
```

Reply:
Ah, good catch!